miio/dreamevacuum_miot.py

```diff
@@ -281,7 +287,7 @@ def get_properties_for_mapping(self, *, max_properties=15) -> list:
         Method was copied from the base class to change the value of max_properties to
         10. This change is needed to avoid "Checksum error" messages from the device.
 
-        # TODO: miotdevice class should have a possibility to define its max_properties value
+	 #TODO: MiotDevice class should have a possibility to define its max_properties value
```

Comment:
```suggestion
	  #TODO: MiotDevice class should have a possibility to define its max_properties value
```
The start of the multiline docstring line must be indented at the same level as the docstring itself.